src/scale.ts

```diff
@@ -347,6 +347,15 @@ export interface ScaleConfig {
    * @minimum 0
    */
   maxStrokeWidth?: number;
+
+  /**
+   * Default range cardinality for discretizing scales, e.g. [`quantile`](https://vega.github.io/vega-lite/docs/scale.html#quantile), [`quantize`](https://vega.github.io/vega-lite/docs/scale.html#quantize), [`threshold`](https://vega.github.io/vega-lite/docs/scale.html#threshold)
```

Comment:
1) The default count of threshold must depend on threshold
2) If the name is quantile, we shouldn't use it for all of discretizing scales.  We can't name it discretizing either as this number won't work for threshold. So we better separate quantile/quantize.

Reply:
Done